### ISSUE TITLE
Update to RN 0.25 to avoid deprecation warning

### ISSUE THREAD

**Issue by exoer** [jineshshah36/react-native-nav#6]
Than you for great components. It would be good if the import statements were updated to comply with react native 0.25 so that we avoid the deprecation warnings. 

Let me know if you want me to do it and submit a pull request. 

```
import React from 'react';

import {
    Text,
    View,
    StyleSheet,
} from 'react-native';
```

**Comment by jineshshah36** [comment]
Could you submit a pull request? Thanks for bringing this to my attention.

**Comment by jineshshah36** [comment]
merged, thanks

**Comment by jineshshah36** [comment]
released in v1.1.3
